docs/commands/browser/runStep.mdx

```diff
@@ -0,0 +1,67 @@
+# runStep
+
+## Обзор {#overview}
+
+Используйте команду `runStep`, чтобы получить человекочитаемую историю выполнения теста, которая, в том числе, автоматически будет отображаться в [html-reporter][reporter].
+Шаги могут быть вложенными.
+
+## Использование {#usage}
+
+```javascript
+await browser.runStep(stepName, stepCb);
+```
+
+## Параметры команды {#parameters}
+
+<table>
+<thead>
+<tr><td>**Имя**</td><td>**Тип**</td><td>**Описание**</td></tr>
+</thead>
+<tbody>
+<tr><td>stepName</td><td>String</td><td>Название шага.</td></tr>
+<tr><td>stepCb</td><td>Function</td><td>Функция с набором команд, которые нужно объединить в единый шаг.</td></tr>
+
+</tbody>
+</table>
+
+## Примеры использования {#examples}
+
+```javascript
```

Comment:
Let's use typescript in all code blocks?

It won't change a thing here, I know, but I see two reasons why it will be useful:
- When we enable that plugin for automatic Typescript=>JS translation of code blocks, we'll get nice&consistent experience across the whole site. If we keep js blocks, they will be the only ones without the TS/JS switch.
- Anyone in the future will be able to add types / typescript specific syntax